shared_governance_base/security/shared_governance_security.xml

```diff
@@ -1,26 +1,27 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8" ?>
 <!--
      Copyright 2019 Coop IT Easy
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 -->
 <odoo>
-    <data>
-        <record id="module_category_shared_governance" model="ir.module.category">
-            <field name="name">Shared Governance</field>
-            <field name="sequence">3</field>
-        </record>
+    <record id="module_category_shared_governance" model="ir.module.category">
+        <field name="name">Shared Governance</field>
+        <field name="sequence">3</field>
+    </record>
 
-        <record id="group_shared_governance_user" model="res.groups">
-            <field name="name">User</field>
-            <field name="implied_ids" eval="[(4, ref('base.group_user'))]"/>
-            <field name="category_id" ref="module_category_shared_governance"/>
-        </record>
+    <record id="group_shared_governance_user" model="res.groups">
+        <field name="name">User</field>
+        <field name="implied_ids" eval="[(4, ref('base.group_user'))]" />
+        <field name="category_id" ref="module_category_shared_governance" />
+    </record>
 
-        <record id="group_shared_governance_coordinator" model="res.groups">
-            <field name="name">Coordinator</field>
-            <field name="implied_ids" eval="[(4, ref('group_shared_governance_user'))]"/>
-            <field name="users" eval="[(4, ref('base.user_root'))]"/>
-            <field name="category_id" ref="module_category_shared_governance"/>
-        </record>
-    </data>
+    <record id="group_shared_governance_coordinator" model="res.groups">
+        <field name="name">Coordinator</field>
+        <field name="implied_ids" eval="[(4, ref('group_shared_governance_user'))]" />
+        <field
+            name="users"
+            eval="[(4, ref('base.user_root')), (4, ref('base.user_admin'))]"
```

Comment:
Only`base.user_admin` is needed here. From 9.0 to 12.0, `base.user_root` was split into `base.user_root` and `base.user_admin`.